rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp

```diff
@@ -814,6 +814,7 @@ rmf_fleet_msgs::msg::RobotState convert_state(const TaskManager& mgr)
         .x(p.x())
         .y(p.y())
         .yaw(l.orientation())
+        .approach_speed(0.0)
```

Comment:
Populating the correct speed should be simple enough to include in this PR?

Reply:
This is actually a bit tricky because the speed limit is currently calculated by knowing the speed limit of the [approach lanes](https://github.com/open-rmf/rmf_traffic/blob/main/rmf_traffic/include/rmf_traffic/agv/Planner.hpp#L738) which are a property of a `Plan` but not of the normal `Graph`, that this function uses to populate the state, at least as far as I understand